src/components/FormDropdown/FormDropdown.js

```diff
@@ -55,104 +21,118 @@ export const FormDropdown = (props) => {
+                : props.dropdownProps.defaultValue
+            }
+            minValue={props.dropdownProps.minValue}
+            maxValue={props.dropdownProps.maxValue}
+            options={props.dropdownProps.options}
+            hasSearch={props.dropdownProps.hasSearch}
+            hasError={props.dropdownProps.hasError}
+            errorText={props.dropdownProps.errorText}
+            onChange={props.dropdownProps.onChange}
           />
-        </button>
-        <div className="dropdown-open">
-          {isOpen && (
-            <ul className="dropdown-options">
-              {props.options
-                .filter((option) =>
-                  option.value
-                    .toLowerCase()
-                    .includes(searchOption.toLowerCase())
-                )
-                .map((option, index) => (
-                  <li
-                    key={option.id}
-                    className={`${
-                      index === 0
-                        ? "ds-border-none"
-                        : "ds-border-t-[1px] ds-border-[#666666] ds-border-opacity-60"
-                    } ${option.value === selectedOption ? "selected" : ""}`}
-                  >
-                    <button
-                      onClick={() =>
-                        props.monthDropdown
-                          ? handleOptionClickMonth(option.value)
-                          : handleOptionClick(option.value)
-                      }
-                      className="dropdown-option"
-                    >
-                      {option.value}
-                    </button>
-                  </li>
-                ))}
-            </ul>
-          )}
         </div>
-        {props.hasError && <FormError errorMessage={props.errorText} />}
-      </div>
+      )}
     </>
   );
 };
 
+FormDropdown.defaultProps = {
+  width: "standard",
+};
+
 FormDropdown.propTypes = {
-  options: PropTypes.arrayOf(
-    PropTypes.shape({ id: PropTypes.string, value: PropTypes.string })
-  ),
-  defaultValue: PropTypes.string,
-  disabled: PropTypes.bool,
-  onChange: PropTypes.func,
   /**
-   * The textfield label
+   * Select the width of the dropdown.
+   */
+  width: PropTypes.oneOf(["standard", "number"]),
+  /**
+   * Determines if the dropdown has a label
    */
   hasLabel: PropTypes.bool,
   /**
-   * Has errors
+   * Dropdown props for a single dropdown
    */
-  hasError: PropTypes.bool,
+  dropdownProps: PropTypes.shape({
+    defaultValue: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    onChange: PropTypes.func,
+    options: PropTypes.arrayOf(
+      PropTypes.shape({ id: PropTypes.string, value: PropTypes.string })
+    ),
+    hasError: PropTypes.bool,
+    errorText: PropTypes.string,
+    minValue: PropTypes.number,
+    maxValue: PropTypes.number,
+    hasSearch: PropTypes.bool,
+  }),
+  /**
+   * Dropdown props for the second dropdown in case of doubleDropdown
+   */
+  dropdown2Props: PropTypes.shape({
```

Comment:
Is 2 the maxunm number of Dropdowns that will ever exist in this component?

Reply:
The double dropdown is built specifically for the OAS estimator team, other people might use this pattern in the future, but for now, it's made for OAS team's need. 
If user want more dropdowns, they can just add more single dropdowns to their app.